low-level/files/opt/seagate/sspl/setup/manifest_support_bundle

```diff
@@ -23,39 +23,39 @@ import sys
 import socket
 import json
 import subprocess
+import tempfile
 
-from cortx.utils.conf_store import Conf
-from framework.utils.conf_utils import Utility
-from framework.base.sspl_constants import (PRODUCT_FAMILY,
-                                        RESOURCE_HEALTH_VIEW,
-                                        sspl_config_path,
-                                        GLOBAL_CONFIG_INDEX)
+from framework.utils.conf_utils import (GLOBAL_CONF, Conf,
+    STORAGE_TYPE_KEY, SITE_ID_KEY, RACK_ID_KEY,
+    NODE_ID_KEY, CLUSTER_ID_KEY)
+from framework.base.sspl_constants import PRODUCT_FAMILY, DATA_PATH
+from framework.platforms.realstor.realstor_enclosure import singleton_realstorencl
```

Comment:
Import singleton_realstorencl only if SSPL state is active. add this import in the capture_enclosure_data method and remove it from here.

Reply:
Agree. Done the changes.